docs/TuxOwneeClass.md

```diff
@@ -0,0 +1,249 @@
+
+##### <a id="OwneeClass-nearestOwnerProps"></a>Property - `OwneeClass.nearestOwnerProps` [#](#OwneeClass-nearestOwnerProps)
+Allows the `Ownee` to access the `nearestOwnerProps` on the closest `OwnerClass`.
+The `nearestOwnerProps` should be static, if you need to pass in dynamic `props` use the standard `React` `props` pass down syntax.
+
+**NOTE** `nearestOwnerProps` should be treated as immutable just like standard component props. Thus, you should never modify the `nearestOwnerProps` object. It is designed to store static methods and data.
+
+***
+
+##### <a id="TuxOwneeClass-Guidelines"></a>4) TuxOwneeClass Guidelines [#](#TuxOwneeClass-Guidelines)
+It is quite possible to never take advantage of the versatility that is provided by using the `nearestsOwnerProps`. Yet you would be missing out on the benefits that it provides. Passing properties down through a multitude of components is error prone, as it is easy to forget to pass all properties or methods that you need. Part of this difficulty can be alleviated by using `propTypes` to validate a components prop inputs and using the `React Spread` syntax to cascade down properties. However, even with those tools prop sharing is difficult and fragile. Additionally, it significantly hurts our ability to reuse components since our components now have to request `props` that they would not normally need just so that they can share those `props` with their children. Finally, cascading down `props` makes your `React` components vulnerable to refactoring/changes in component structure since you have to make sure you have the correct `props` in the correct places whenever you want to restructure your components. With `Tux`, as long as the 'nearestOwnerClasss` to the component doesn't change you can freely refactor your `React` structure without having to worry about losing access to your needed properties.
+
+Using `nearestOwnerProps` allows you to know exactly what is available to an `OwneeClass`, without having to muddle through figuring out what will still be on that `props` object that you passed through `React's` `Spread` syntax. Therefore if you are passing methods from one child to the next only because you must expose those methods to their children, we recommend utilizing the `nearestOwnerProps` on the `OwneeClass`. It exposes the methods and properties of an `OwnerClass` via symantic and easy to reason about code.
+
+```javascript
+    //BAD
+    var React = require('tux/React');
+
+    // Passing properties down using the standard React prop syntax can be error prone, as you will have to manage and track sharing your methods throughout your component hierarchy.
+    var Message = React.createOwneeClass({
+      render: function () {
+        return (
+          <div>
+            <p>{this.props.message.text}</p>
```

Comment:
Add brief comment explaining why we still want to read the message from `props`.